utils/build-script-impl

```diff
@@ -990,20 +990,22 @@ fi
 if [[ ! "${SKIP_BUILD_LLBUILD}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" llbuild)
 fi
-if [[ ! "${SKIP_BUILD_SWIFTPM}" ]] ; then
-     PRODUCTS=("${PRODUCTS[@]}" swiftpm)
-fi
 if [[ ! "${SKIP_BUILD_LIBDISPATCH}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" libdispatch)
 fi
-# XCTest has a dependency on Foundation, so Foundation must be added to the
-# list of build products first.
+# SwiftPM and XCTest are dependent on Foundation, so Foundation must be
+# added to the list of build products first.
 if [[ ! "${SKIP_BUILD_FOUNDATION}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" foundation)
 fi
+# SwiftPM is dependent on XCTest, so XCTest must be added to the list of
```

Comment:
I don't think SwiftPM actually requires XCTest as a build-time dependency. I would actually prefer to see SwiftPM built first to facilitate migrating XCTest to be built by SwiftPM in the future.